README.md

```diff
@@ -1,98 +1,101 @@
    will build the project, and run tests and validation on the
    packer configuration.  It will __not__ build an image.
 1. **Pre-release deploy**: Publish a GitHub release
    with the "This is a pre-release" checkbox checked.  An image will be built
-   and deployed to the single region defined by the `PACKER_BUILD_REGION`
-   environment variable.
+   and deployed using the [`prerelease`](.github/workflows/prerelease.yml)
+   workflow.  This should be configured to deploy the image to a single region
+   using a non-production account.
 1. **Production release deploy**: Publish a GitHub release with
    the "This is a pre-release" checkbox unchecked.  An image will be built
-   in the `PACKER_BUILD_REGION` and copied to each region listed in the
-   `PACKER_DEPLOY_REGION_KMS_MAP` environment variable.
+   and deployed using the [`release`](.github/workflows/release.yml)
+   workflow.  This should be configured to deploy the image to multiple regions
+   using a production account.
 
 ### Using Your Local Environment ###
 
-The following environment variables are used by Packer:
+Packer will use your
+[standard AWS environment](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)
+to build the image.
+
+The [Packer template](src/packer.json) requires one environment variable to be defined:
+
+- `BUILD_REGION`: the region in which to build the image.
+
+Additionally, the following optional environment variables can be used
+by the [Packer template](src/packer.json) to tag the final image:
 
-- Required
-  - `PACKER_BUILD_REGION`: the region in which to build the image.
-  - `PACKER_DEPLOY_REGION_KMS_MAP`: a map of deploy regions to KMS keys.
-- Optional
-  - `GITHUB_ACCESS_TOKEN`: a personal GitHub token to use for API access.
-  - `PACKER_IMAGE_VERSION`: the version tag applied to the final image.
+- `GITHUB_IS_PRERELEASE`: boolean pre-release status
+- `GITHUB_RELEASE_TAG`: image version
+- `GITHUB_RELEASE_URL`: URL pointing to the related GitHub release
 
 Here is an example of how to kick off a pre-release build:
 
 ```console
 pip install --requirement requirements-dev.txt
-export PACKER_BUILD_REGION="us-east-2"
-export PACKER_DEPLOY_REGION_KMS_MAP="us-east-1:alias/cool/ebs,us-east-2:alias/cool/ebs,us-west-1:alias/cool/ebs,us-west-2:alias/cool/ebs"
-export PACKER_IMAGE_VERSION=$(./bump_version.sh show)
 ansible-galaxy install --force --force-with-deps --role-file src/requirements.yml
-./patch_packer_config.py pre-release src/packer.json
+export BUILD_REGION="us-east-2"
+export GITHUB_RELEASE_TAG=$(./bump_version.sh show)
+echo "us-east-2:alias/cool/ebs" | ./patch_packer_config.py src/packer.json
 packer build --timestamp-ui src/packer.json
 ```
 
 If you are satisfied with your pre-release image, you can easily create a release
-that deploys to all regions by changing the `pre-release` command of
-`patch_packer_config.py` to `release` and rerunning packer:
+that deploys to all regions by changing the input to
```

Comment:
Are you talking about deploying a pre-release to all regions?  Or are you talking about a real release here?  This section is a little confusing because your examples don't show the value of the `GITHUB_IS_PRERELEASE` env variable.

Reply:
This section is not about using GitHub to do true pre-releases, and releases.  This is hand-jamming it on your local box (for @jsf9k) .  The `GITHUB_IS_PRERELEASE` variable is optional and only used to tag the AMI.

Reply:
I recommend switching to the side-by-side diff view to more easily compare the two documents as wholes instead of intermingled.